docs/rebuild_admin.rst

```diff
@@ -0,0 +1,404 @@
+reinstalling the instance may be simpler. An outline of the steps involved is as 
+follows:
+
+ #. Prepare the USB sticks.
+ #. (Optional) Boot the *Application* and *Monitor Server* in single user mode and  reset
+    the shell admin account password.
+ #. Set up SSH access for the new *Admin Workstation*.
+ #. Retrieve SecureDrop configuration settings from the  *Application* and *Monitor Server*.
+ #. Back up and configure the SecureDrop application.
+ #. Run the ``install`` and ``tailsconfig`` commands from the new *Admin Workstation*.
+ #. Complete post-rebuild tasks.
+
+
+Step 1: prepare USB sticks
+==========================
+
+First, create a Tails Master USB using the installation procedure described here: 
+https://tails.boum.org/install/index.en.html
+
+Next, use the Tails Master USB to :ref:`create a new Admin Workstation USB <set_up_tails>` and set up a persistent volume with a strong passphrase.
+
+Once persistence has been set up, start up the Admin Workstation with 
+persistence enabled, :ref:`install the SecureDrop application code, and set up
+the KeePassX database <set_up_admin_tails>`.
+
+The Admin Workstation uses SSH with key authentication to connect to the servers,
+so you'l need to create a new SSH keypair for your SecureDrop instance. To do so,
+open a Terminal and run the following command:
+
+.. code:: sh
+
+ ssh-keygen -t rsa -b 4096
+
+When prompted to ``Enter file in which to save the key``, Press **Enter** to use
+the default location.
+
+
+Step 2: (Optional) Boot the servers in single-user mode
+=======================================================
+If you do not have the original password for the shell admin account on the
+*Apllication* and *Monitor Servers*, you'll need to reset the password on each
+server by booting in single user mode. In order to do so, you'll need physical
+access to the server, a keyboard, and a monitor. 
+
+First, connect a monitor and keyboard to the *Monitor Server*. Then reboot the server.
+When the GRUB menu appears, make sure the **Ubuntu** entry is highlighted, and 
+press **e** to edit boot options.
+In the boot options for Ubuntu, find the line that starts with ``linux`` and ends
+with ``ro``. Add `` single`` after ``ro`` (make sure to include the space!) and press
+**F10** to boot in single user mode.
+
+Reset the SecureDrop admin user's password
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Once the root prompt appears, you'll need to reset the password for the
+SecureDrop admin user. By default this user is named `sdadmin` and has UID 1000.
+However it may have been named differently during the installation of your 
+instance. You can use the command ``getent passwd 1000`` to check the username
+corresponding to UID 1000. Once you have the correct username, reset its password
+using the `passwd` command, for example:
+
+.. code:: sh 
+
+ passwd sdadmin
+
+.. important::
+ Make sure to select a strong password!
```

Comment:
I'd suggest that we recommend generating and storing the password (or *passphrase*; see #2176) using KeePassX on the Admin Workstation, especially since we just instructed them to configure it.

Reply:
done.